woocommerce/wc-admin
# Update sorted column background color

Fixes #448 

Update column color in tables to light gray when sorted.  Hover color should override this color.

### Screenshots

<img width="833" alt="screen shot 2018-10-18 at 9 19 12 am" src="https://user-images.githubusercontent.com/10561050/47157245-eacd3d00-d2b6-11e8-8cd9-fdcc051f6f7c.png">
<img width="848" alt="screen shot 2018-10-18 at 9 19 19 am" src="https://user-images.githubusercontent.com/10561050/47157246-eb65d380-d2b6-11e8-8ed9-36b79196efcb.png">

### Detailed test instructions:

1.  Visit `/wp-admin/admin.php?page=wc-admin#/analytics/revenue`
2.  Click on any column header to sort that column
3.  Confirm that background-color of the sorted column is light gray
4.  Confirm that hovering the column overrides the sorted color with the hovered row color
